frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandlerTest.java

```diff
@@ -558,6 +559,38 @@ void shouldChangeStateToAccountTemporarilyLockedAfter5UnsuccessfulAttempts(
                                                 .equals(expectedCommonSubject)));
     }
 
+    @ParameterizedTest
+    @EnumSource(MFAMethodType.class)
+    void shouldChangeStateToAccountTemporarilyLockedAfter5UnsuccessfulAttemptsForReauthJourney(
+            MFAMethodType mfaMethodType) {
```

Comment:
Just a check since - do we have a happy path test for the reauth journey i.e. logging in successfully? Wondering if there's already a test that goes through every `JourneyType` and does the success case but haven't read through this whole test file 😂

Reply:
The reauth journey type is being covered by `shouldChangeStateToAccountTemporarilyLockedAfter5UnsuccessfulAttemptsForReauthJourney` test. The other journey type that aligns with the Login handler is `SIGN_IN`. There isn't any use of it in the Login handler so don't think it requires a test.